.github/workflows/build_x86_64.yml

```diff
@@ -0,0 +1,94 @@
+#=================================================
+# Description: Test build OpenWrt using GitHub Actions
+# Lisence: MIT
+# Author: Xiaobo Tian peterwillcn@gmail.com
+#=================================================
+
+name: Build_x86_64
+
+on: [push, pull_request]
+#  push:
+#    branches:
+#      - master
+
+  #schedule:
+  #  - cron: 0 8 * * 5
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@master
```

Comment:
Do I understand correctly that you test the master branch on pull request ?

Reply:
Also you don't seem to use this checkout at all

Reply:
You're right, it's not really used here.